tests/Hedgehog.Tests/Hedgehog.Tests.fsproj

```diff
@@ -12,10 +12,22 @@
     <Compile Include="MinimalTests.fs" />
   </ItemGroup>
   <ItemGroup>
-    <None Include="paket.references" />
+    <PackageReference Include="Microsoft.NET.Test.Sdk" Version="16.6.1" />
+    <PackageReference Include="Unquote" Version="4.0.0" />
+    <PackageReference Include="xunit" Version="2.4.1" />
+    <PackageReference Include="xunit.runner.console" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.4.1">
+      <PrivateAssets>all</PrivateAssets>
+      <IncludeAssets>runtime; build; native; contentfiles; analyzers; buildtransitive</IncludeAssets>
+    </PackageReference>
   </ItemGroup>
   <ItemGroup>
     <ProjectReference Include="..\..\src\Hedgehog\Hedgehog.fsproj" />
   </ItemGroup>
-  <Import Project="..\..\.paket\Paket.Restore.targets" />
+  <ItemGroup>
+    <PackageReference Update="FSharp.Core" Version="4.2.3" />
```

Comment:
See comment above about `Update`. And perhaps we want `Version="4.*"`?

Reply:
`4.*` probably isn't right because `FSharp.Core` version `4.7.0` doesn't support .NET Core 2.0 but version `4.6.2` does

Reply:
It is ok to depend on a specific version `FSharp.Core` here because this project isn't a library project consumed by others but an executable project only used "internally" by the "Hedgehog developers" for testing.  

paket's lock file shows that it resolved version 4.6.1 for the `Test` group.  I picked version 4.2.3 because it matches the constraint imposed by Unqoute before I thought to look at the lock file.  I think picking any of these versions is fine.
https://github.com/hedgehogqa/fsharp-hedgehog/blob/2a68ee22e85b5d9026b3b21a6a0a04bce9cef086/paket.lock#L475